test/packaging/windows/msi.tests.ps1

```diff
@@ -74,13 +74,9 @@ Describe -Name "Windows MSI" -Fixture {
     }
     AfterEach {
         if ($Error.Count -ne 0 -and !$uploadedLog) {
-            if ($env:APPVEYOR) {
-                Push-AppveyorArtifact $msiLog
```

Comment:
I added a test helper for this, with the conditionals in the helper.

Reply:
Looks like it will work